extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -116,12 +121,24 @@ export class AmpAdApiHandler {
         (waitForRenderStart.indexOf(this.baseInstance_.adType) >= 0);
 
     if (renderStartImplemented) {
+      const perf = performanceFor(this.baseInstance_.win);
+      const initTime = Date.now();
+      const adTag = adTypeAbbrevTable[this.baseInstance_.adType];
+      // Report performance for all renderStartImplemented cases.
+      // This is needed to calculate the perc of receiving nothing from ad.
+      perf.tickDelta(adTag, 0);
+      perf.flush();
       // If support render-start, create a race between render-start no-content
       this.adResponsePromise_ = listenForOncePromise(this.iframe_,
         ['render-start', 'no-content'], this.is3p_).then(info => {
           if (info.data.type == 'render-start') {
```

Comment:
As discussed in one of the email thread, we are not interested (and probably should not) to differentiate `render-start` and `no-content`. Instead, we would rather report the ad network name. 

@erwinmombay is there a way to attach the name so that CSI report can break down on ad network name?